parcels/field.py

```diff
@@ -281,16 +281,17 @@ def from_netcdf(cls, name, dimensions, filenames, indices={},
         timeFiles = []
         for fname in filenames:
             with NetcdfFileBuffer(fname, dimensions, indices) as filebuffer:
-                timeslices.append(filebuffer.time)
-                timeFiles.append([fname for i in range(len(filebuffer.time))])
+                ftime = filebuffer.time
+                timeslices.append(ftime)
+                timeFiles.append([fname for i in range(len(ftime))])
```

Comment:
Since `i` is not actually used, would it be better to do `[fname for _ in range(len(ftime))]`? Would that work?